src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftExcludeTranspilation.md

```diff
@@ -0,0 +1,254 @@
+This document goes over one of the more complicated rewrites for PartiQL's `EXCLUDE` to Redshift.
```

Comment:
Would recommend for any reviewers of the PR to start here to understand the PartiQL -> Redshift rewrite and the Redshift `OBJECT_TRANSFORM` function.